docs/src/misc.rst

```diff
@@ -378,3 +378,24 @@ API
         stability guarantees.
 
     .. versionadded:: 1.8.0
+
+.. c:function:: int uv_randbytes(void* buf, size_t len)
+
+    Fill the given buffer with cryptographically suitable random data from the system CSPRNG.
+    This function makes no guarantees about blocking or not doing so.  Likewise, the output
+    random data is not guaranteed to be cryptographically suitable for all platforms, iff
```

Comment:
Hmm, I've never seen that used anywhere before. Would it not be clearer to just say "if and only if"?